### PR TITLE
Allow to edit loadout name via clicking on loadout name

### LINKED ISSUE #2135: Edit Loadout name/desc by clicking on loadout title
Similar to https://github.com/frzyc/genshin-optimizer/issues/2134

image

### PR DESCRIPTION
**Pull request by bluezettabyte**
## Describe your changes

Allow users to click on loadout name to edit loadout name/description

## Issue or discord link

Resolve #2135 

## Testing/validation

image

## Checklist before requesting a review (leave this PR as draft if any part of this list is not done.)

- [ ] I have commented my code in hard-to understand areas.
- [ ] I have made corresponding changes to README or wiki.
- [ ] For front-end changes, I have updated the corresponding English translations.
- [x] I have run `yarn run mini-ci` locally to validate format and lint.
- [ ] If I have added a new library or app, I have updated the deployment scripts to ignore changes as needed

### CODE REVIEW

File: libs/gi/page-team/src/CharacterDisplay/Content.tsx [hunk 1]
```diff
@@ -122,34 +140,73 @@ function TabNav({
...
+                    display="flex"
+                    flexDirection="column"
+                    gap={2}
+                    sx={{ mt: 2 }}
+                  >
+                    <TextFieldLazy
+                      label="Loadout Name"
+                      placeholder='Loadout Name'
+                      value={teamChar.name}
+                      onChange={(loadoutName) => handleName(loadoutName)}
```
Comment (frzyc) [inline libs/gi/page-team/src/CharacterDisplay/Content.tsx:194]: more of a nit, can be simplified
```suggestion
                      onChange={handleName}
```

**Review by frzyc — CHANGES_REQUESTED** [review]
LGTM, just need to pass format/lint.

File: libs/gi/page-team/src/CharacterDisplay/Content.tsx [hunk 2]
```diff
@@ -131,24 +155,68 @@ function TabNav({
             position: 'relative',
```
Comment (nguyentvan7) [inline libs/gi/page-team/src/CharacterDisplay/Content.tsx:155]: Change the `pb` above to `p`

File: libs/gi/page-team/src/CharacterDisplay/Content.tsx [hunk 3]
```diff
@@ -131,24 +155,68 @@ function TabNav({
...
-            title={
-              teamChar.description ? (
-                <Typography>{teamChar.description}</Typography>
-              ) : undefined
-            }
-          >
-            <Typography
-              variant="h6"
-              sx={{ display: 'flex', gap: 1, alignItems: 'center' }}
+          <CardActionArea onClick={() => setEditMode(true)}>
```
Comment (nguyentvan7) [inline libs/gi/page-team/src/CharacterDisplay/Content.tsx:158]: add `sx={{ p: 2 }}`

**Review by nguyentvan7 — CHANGES_REQUESTED** [review]
Padding fix as well, sorry for iterations

**Comment by nguyentvan7** [comment]
Also format failed

**Review by frzyc — APPROVED** [review]
LGTM

**Pull request merged**